src/controllers/ColorStyleController.ts

```diff
@@ -0,0 +1,101 @@
+import { EditorAPI } from '../../types/CommonTypes';
+import { ColorUpdate } from '../../types/ColorStyleTypes';
+
+/**
+ * The ColorStyleController is responsible for all communication regarding color styles.
+ * Methods inside this controller can be called by `window.SDK.colorStyle.{method-name}`
+ */
+export class ColorStyleController {
+    /**
+     * @ignore
+     */
+    #editorAPI: EditorAPI;
+
+    /**
+     * @ignore
+     */
+    constructor(editorAPI: EditorAPI) {
+        this.#editorAPI = editorAPI;
+    }
+
+    /**
+     * This method returns the list of colors
+     * @returns
+     */
+    getColors = async () => {
+        const res = await this.#editorAPI;
+        return res.getColors();
+    };
+
+    /**
+     * This method returns a color by id
+     * @param colorId The ID of a specific color
+     * @returns
+     */
+    getColorById = async (colorId: string) => {
+        const res = await this.#editorAPI;
+        return res.getColorById(colorId);
+    };
+
+    /**
+     * This method create a new color
+     * @returns the new created color id
+     */
+    createColor = async () => {
+        const res = await this.#editorAPI;
+        return res.createColor();
+    };
```

Comment:
shouldn't we pass the color as param?

Reply:
there is no param on the methods exposed by Flutter